File: Java/src/main/java/com/gildedrose/ConjuredItem.java
package com.gildedrose;

/**
 *
 * @author markd
 */
public class ConjuredItem extends GenericItem{
    
    public ConjuredItem(String name, int sellIn, int quality) {
        super(name, sellIn, quality);
        QUALITY_CHANGE_BEFORE_SELLBY = -2;
        QUALITY_CHANGE_AFTER_SELLBY = -4;
    }
    
}

File: cpp-catch2/src/ConjuredItem.cpp
//
// Created by markd on 3/9/2021.
//

#include "ConjuredItem.h"

ConjuredItem::ConjuredItem(string name, int sellIn, int quality) : GenericItem(name, sellIn, quality) {
    QUALITY_CHANGE_BEFORE_SELLBY = -2;
    QUALITY_CHANGE_AFTER_SELLBY = -4;
}
